src/store/reducers/schema.ts

```diff
@@ -204,14 +204,20 @@ export const selectSchemaChildren = (state: ISchemaRootStateSlice, path: string
 export const selectSchemaData = (state: ISchemaRootStateSlice, path: string | undefined) =>
     path ? state.schema.data[path] : undefined;
 
-export const selectSchemaChildrenPaths: Selector<
+export const selectSchemaMergedChildrenPaths: Selector<
     ISchemaRootStateSlice,
     string[] | undefined,
-    [string | undefined]
+    [string | undefined, boolean]
 > = createSelector(
-    [(_, path: string | undefined) => path, selectSchemaChildren],
-    (path, children) => {
-        return children?.map(({Name}) => path + '/' + Name);
+    [
+        (_, path?: string) => path,
+        (_, _path, isEntityWithMergedImplementation: boolean) => isEntityWithMergedImplementation,
+        selectSchemaChildren,
+    ],
+    (path, isEntityWithMergedImplementation, children) => {
+        return isEntityWithMergedImplementation
```

Comment:
I can import check from [schema](https://github.com/ydb-platform/ydb-embedded-ui/blob/06d26def15496f8e2de00d941b39bf6a68382f14/src/containers/Tenant/utils/schema.ts) and pass the type here. But it will be an import from components. How do you think, where should I place schema, to keep imports more clear?